### PR TITLE
Cameraroll v1.5.2

### PR DESCRIPTION
**Pull request by deepueg**
Please review the last commit.

### CODE REVIEW

File: plugins/ern_v0.14.0+/@react-native-community/cameraroll_v1.1.2+/config.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "android": {
+    "root": "android",
+    "dependencies": []
```
Comment (krunalsshah) [inline plugins/ern_v0.14.0+/@react-native-community/cameraroll_v1.1.2+/config.json:4]: missing permissions <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>

File: plugins/ern_v0.14.0+/rn-fetch-blob_v0.12.0+/config.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
+{
+  "android": {
+    "root": "android",
+    "dependencies": []
```
Comment (krunalsshah) [inline plugins/ern_v0.14.0+/rn-fetch-blob_v0.12.0+/config.json:4]: permissions

```
+   <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
+   <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
+   <uses-permission android:name="android.permission.DOWNLOAD_WITHOUT_NOTIFICATION" />
```

Comment (deepueg) [reply plugins/ern_v0.14.0+/rn-fetch-blob_v0.12.0+/config.json]: Added a README for this as these permissions needed to be added based on what features they are going to use.

File: plugins/ern_v0.14.0+/@react-native-community/cameraroll_v1.1.2+/config.json (hunk 1 above)
Comment (deepueg) [reply plugins/ern_v0.14.0+/@react-native-community/cameraroll_v1.1.2+/config.json]: Done

**Pull request merged**